contrib/ansible/create-cluster-playbook.yaml

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/ansible-playbook
+#
+# Create a test Cluster in the Cluster Operator.
+#
+---
+- hosts: localhost
+  connection: local
+  gather_facts: no
+
+  # Variables you can optionally override from the CLI:
+  vars:
+    # NOTE: you may set 'cluster_namespace' to control where the cluster object is created. We do not
+    # set a default here as we need to lookup the current one if unset.
+
+    # ID for your cluster, defaults to current username:
+    cluster_name: "{{ lookup ('env', 'USER') }}"
+
+    # ClusterVersion to install, default to fake so we don't actually create a cluster unless user
```

Comment:
s/ClusterVersion to install/ClusterVersion to use/
You're not installing a ClusterVersion